Description: test_paths: skip tests which assume non-system python
 See https://github.com/jupyter/jupyter_core/issues/208
 Not forwarded as this does not fix the tests themselves
Forwarded: not-needed
Author: Gordon Ball <gordon@chronitis.net>
Last-Update: 2021-11-02

--- a/tests/test_paths.py
+++ b/tests/test_paths.py
@@ -33,6 +33,8 @@
     jupyter_runtime_dir,
     prefer_environment_over_user,
     secure_write,
+    SYSTEM_JUPYTER_PATH,
+    ENV_JUPYTER_PATH,
 )
 
 pjoin = os.path.join
@@ -283,6 +285,7 @@
         assert path[-2:] == system_path
 
 
+@pytest.mark.skipif(set(ENV_JUPYTER_PATH) < set(SYSTEM_JUPYTER_PATH), reason="assumes non-system python")
 def test_jupyter_path_user_site():
     with patch.object(site, "ENABLE_USER_SITE", True):
         path = jupyter_path()
@@ -301,6 +304,7 @@
             assert p == v
 
 
+@pytest.mark.skipif(set(ENV_JUPYTER_PATH) < set(SYSTEM_JUPYTER_PATH), reason="assumes non-system python")
 def test_jupyter_path_no_user_site():
     with patch.object(site, "ENABLE_USER_SITE", False):
         path = jupyter_path()
@@ -308,6 +312,7 @@
         assert path[1] == paths.ENV_JUPYTER_PATH[0]
 
 
+@pytest.mark.skipif(set(ENV_JUPYTER_PATH) < set(SYSTEM_JUPYTER_PATH), reason="assumes non-system python")
 def test_jupyter_path_prefer_env():
     with prefer_env:
         path = jupyter_path()
